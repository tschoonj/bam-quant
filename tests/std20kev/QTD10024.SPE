$DATA:
0 2000
20943
13143
5719
1637
328
60
5
6
1
0
1
1
1
0
0
1
110
702
767
676
730
661
614
640
619
601
613
526
553
533
564
519
523
527
482
424
500
436
435
465
432
493
412
427
423
391
389
414
429
389
370
384
333
376
383
421
390
385
392
408
371
400
356
359
361
355
361
347
339
337
383
343
354
349
348
319
325
326
296
319
324
313
315
338
301
302
297
317
325
341
366
326
344
307
360
352
317
318
378
355
337
364
349
376
363
386
342
352
328
361
291
333
323
353
329
340
347
330
290
296
312
325
305
313
332
331
320
310
332
366
330
331
324
335
317
338
308
362
311
324
322
332
330
378
457
547
522
537
506
463
452
387
288
327
360
352
353
355
382
391
379
352
312
362
349
335
354
368
349
340
359
415
407
400
464
519
603
884
1079
1442
1831
2029
1932
1717
1406
1007
691
524
435
447
374
416
431
418
401
416
437
434
450
437
432
483
432
497
562
575
650
674
685
630
615
597
553
521
495
530
495
496
525
550
500
596
543
587
598
560
615
618
610
681
704
671
688
708
722
653
700
730
681
724
708
695
752
797
801
795
889
941
963
1016
1083
1159
1274
1369
1570
1699
2072
2624
3454
5582
10528
21072
41445
76490
126245
187720
248856
294290
306826
283668
233450
169925
110261
63206
32477
15206
6779
3069
1814
1304
1276
1217
1252
1388
1576
2016
2817
4522
7570
12402
19080
27914
36915
44295
47755
46078
40050
31131
21570
13599
7595
3917
1780
738
285
128
62
41
32
39
31
33
39
36
31
41
40
46
42
30
28
20
19
18
17
15
11
11
10
10
9
6
8
9
12
7
9
10
9
10
11
6
12
6
7
16
16
5
14
18
16
8
17
12
11
12
6
7
11
9
6
5
5
12
10
10
14
8
10
9
16
10
13
8
5
13
10
4
9
6
9
6
9
5
6
4
9
9
11
13
4
8
13
5
9
14
7
14
15
13
9
6
9
7
11
1
6
12
8
13
14
4
14
12
16
12
9
10
11
4
12
4
8
11
12
12
5
9
13
17
8
15
11
10
9
13
8
6
7
13
5
9
10
11
13
10
15
19
9
15
16
14
21
15
12
14
13
6
5
9
14
9
5
5
5
5
8
10
10
10
9
6
10
5
15
14
12
14
12
10
10
13
11
14
8
12
5
7
15
8
8
7
12
7
18
9
12
5
4
15
6
11
15
10
14
10
7
9
13
7
11
7
11
20
16
12
8
7
11
11
10
16
10
15
33
24
41
56
99
138
207
282
375
461
557
547
533
527
439
323
280
198
146
76
57
34
28
18
13
19
10
21
22
39
39
61
64
75
121
149
184
175
186
164
150
127
113
57
53
32
25
24
7
14
10
5
5
5
9
5
9
8
15
13
24
12
20
17
20
23
11
13
15
15
5
7
8
5
8
0
2
0
3
2
2
6
2
3
4
5
2
5
2
5
2
0
2
0
6
6
5
6
6
4
5
6
4
1
4
0
7
1
4
1
2
1
4
4
2
3
3
3
0
4
3
3
3
1
1
3
7
0
1
6
1
7
6
3
1
4
1
3
3
1
3
3
4
3
2
2
2
2
1
1
1
2
3
3
0
1
3
1
1
3
2
2
2
1
4
0
2
2
2
0
5
1
3
1
3
2
0
3
3
1
3
1
3
1
5
0
0
0
0
1
3
3
4
0
0
1
1
1
0
0
0
0
1
0
1
3
2
0
1
0
2
0
2
1
2
1
1
3
2
2
0
2
3
4
2
0
2
1
1
1
1
1
2
2
0
2
1
1
1
2
1
2
3
0
6
3
9
6
11
16
19
18
23
34
32
35
50
35
28
28
23
19
18
14
8
5
4
4
0
3
2
1
1
2
3
2
2
5
2
0
3
1
3
3
1
4
1
2
1
2
0
2
2
2
0
0
3
3
1
3
2
1
1
2
2
2
3
2
1
4
3
2
2
5
6
5
2
2
2
2
6
2
0
4
2
1
5
4
4
3
4
8
3
4
2
4
8
8
6
2
9
7
10
7
2
7
6
8
9
7
12
4
13
6
8
5
5
7
4
7
7
7
5
9
7
8
6
5
8
4
5
9
8
9
12
8
10
11
9
10
9
11
7
7
6
10
9
11
14
10
8
12
9
9
8
17
20
15
18
18
16
15
22
14
18
16
23
23
21
16
29
32
28
26
34
36
40
42
43
35
36
44
42
39
53
40
54
48
57
57
51
56
56
56
53
62
58
67
51
65
52
59
66
65
62
58
63
62
66
57
77
70
77
84
89
105
113
113
139
128
132
160
159
154
168
171
167
152
150
142
140
134
122
111
96
80
71
77
51
44
39
39
18
16
16
10
11
6
4
6
3
4
3
1
0
0
1
0
1
0
1
0
1
0
0
0
0
0
1
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
2
0
0
0
1
1
0
0
0
1
1
0
1
0
0
0
0
1
0
0
3
0
0
0
2
0
0
1
0
0
1
0
1
1
1
1
1
1
1
0
0
0
2
0
1
0
1
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10024.spe
Start: 21.07.2014 18:03:35
Stop: 21.07.2014 18:05:16
BESSY: 298.7885774 298.3525223
IONI: 3.934346E-11 3.866063E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 89719
Run: 100000
Dead: 10
ICR: 29240
OCR: 0
Accumulated BESSY: 59675.2921046002
Accumulated Ioni: 7.81285859E-09
N Accumulation: 200
Normfactor:  2.8532223e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cr
 
exp_x 111.260398864746 111.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  24   1      5.412   2239686.      1497.      94.50
