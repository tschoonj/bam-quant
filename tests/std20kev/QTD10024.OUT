     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:38  บ   
     บ  Spectrum: QTD10024.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    58.5   last change =   .04%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -49.0 ๑   100.0   บ     -49.0 ๑      .8   บ   
     บ  GAIN (eV/ch)  บ     20.315 ๑   2.000   บ    20.315 ๑    .003   บ   
     บ det NOISE (eV) บ      -73.4 ๑    40.0   บ     -73.4 ๑    -4.8   บ   
     บ  FANO factor   บ       .220 ๑    .050   บ      .220 ๑    .004   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Cr-Ka                           2239686. ๑   1497.           บ   
     บ    KA1         5.412   1.00000     2239686. ๑   1497.     94.5  บ   
     บ              268.818    141.68         113.           8.68E-01  บ   
     บ                                                                 บ   
     บ  2 Cr-Kb                            370735. ๑    609.           บ   
     บ    KB1         5.947   1.00000      370735. ๑    609.     38.9  บ   
     บ              295.154    150.29         123.           8.98E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                            6714. ๑     85.           บ   
     บ    L1N3       10.824    .73595        4941. ๑     63.     34.6  บ   
     บ              535.225    213.38          76.           9.82E-01  บ   
     บ    L1N2       11.359    .24385        1637. ๑     22.      6.0  บ   
     บ              561.561    219.20          77.           9.84E-01  บ   
     บ    L1O3       11.894    .02020         136. ๑      9.      1.2  บ   
     บ              587.896    224.87          70.           9.86E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
